# The man page change here is Solaris-specific and thus is not being
# contributed back to the upstream community.  The servconf.c change
# was developed in-house and has been submitted upstream to OpenSSH for
# their consideration:
# https://bugzilla.mindrot.org/show_bug.cgi?id=2278
#
# Update - accepted upstream:
# https://github.com/openssh/openssh-portable/commit/ac908c1eeacccfa
#
# OpenSSH normally updates lastlog when users login and reads lastlog to
# determine the last login time of the user however on Solaris the
# pam_unix_session(7) module handles both of these tasks so we compile
# OpenSSH with --disable-lastlog to disable all lastlog references.
# This means the sshd_config option of PrintLastLog is now ignored and
# can't be changed so we update sshd's configuration parsing to flag
# this as unsupported and update the man page here.
#
diff --git a/sshd_config.5 b/sshd_config.5
index 6fa421c..f9ff992 100644
--- a/sshd_config.5
+++ b/sshd_config.5
@@ -1427,8 +1427,8 @@ Specifies whether
 .Xr sshd 8
 should print the date and time of the last user login when a user logs
 in interactively.
-The default is
-.Cm yes .
+On Solaris this option is always ignored since pam_unix_session(7)
+reports the last login time.
 .It Cm PrintMotd
 Specifies whether
 .Xr sshd 8
@@ -1896,7 +1896,8 @@ This file should be writable by root only, but it is recommended
 .El
 .Sh SEE ALSO
 .Xr sftp-server 8 ,
-.Xr sshd 8
+.Xr sshd 8 ,
+.Xr pam_unix_session 7
 .Sh AUTHORS
 .An -nosplit
 OpenSSH is a derivative of the original and free
